src/stores/regulations.ts

```diff
@@ -0,0 +1,50 @@
+import api from '@/utils/api'
+import { defineStore } from 'pinia'
+
+type regulation = { id: string, name: string, slug: string, summary: string | null, text: string, created_at: Date, updated_at: Date }
+
+export const useRegulationsStore = defineStore('regulations', {
+    state: () => {
+        return {
+            regulations: []
+        }
+    },
+    getters: {
+        getRegulation(state: any) {
+            return (slug: string) => {
+                return state.regulations.filter((regulation: any) => regulation.slug === slug)
```

Comment:
get işlemi için şuan tümünü çekmemiz gerekiyor
oysaki bizim zaten tekil olarak ulaşabildiğimiz servisimiz var.

Reply:
evet biliyorum. Olur da bir şekilde almamız gerekir diye kenarda dursun dedim. Zaten kullanmadım